drop-storage/Cargo.toml

```diff
@@ -9,3 +9,4 @@ edition = "2021"
 thiserror = "*"
 sqlx = { version = "0.6", features = [ "runtime-tokio-rustls", "sqlite", "time", "chrono" ] }
 slog = { workspace = true }
+serde = { version = "1.0.160" }
```

Comment:
It's used in at least 2 other crates. I'm suggesting moving it to workspace `Cargo.toml` to avoid duplicate versions